ci/.gitlab-ci.yml

```
@@ -134,6 +156,28 @@ cpu release build clang13 cxx20:
     THREADS_PER_NODE: 72
     USE_CODECOV: "false"
 
+cpu release build clang16 cxx20:
+  extends: .build_spack_common
+  variables:
+    BUILD_DOCKER_FILE: ci/docker/build.Dockerfile
+    DEPLOY_DOCKER_FILE: ci/docker/deploy.Dockerfile
+    BASE_IMAGE: docker.io/ubuntu:23.04
+    DEPLOY_BASE_IMAGE: docker.io/ubuntu:23.04
+    EXTRA_APTGET: clang-16
```

```suggestion
    EXTRA_APTGET: "clang-16 libomp-16-dev"
```
We can do this here or in #668

---

Good point. I think we can have #668 in first. I'll update this here already now.

---

https://github.com/eth-cscs/DLA-Future/pull/853/commits/3ee241fc7295b4b2143e9996b47ff118cfa25efc